everviz/data/load_csv/get_data.py

```diff
@@ -6,4 +6,4 @@
 @CACHE.memoize(timeout=CACHE.TIMEOUT)
 @webvizstore
 def get_data(data_path) -> pd.DataFrame:
-    return pd.read_csv(data_path, delimiter=";")
+    return pd.read_csv(data_path)
```

Comment:
Are we still able to use the everest csv export after switching to the default delimiter for the crossplot plugins? The everest csv file uses the `;` delimiter and I guess that is why it was there in the first place.

Reply:
You´re right, and probably wont have this in the final PR, was just to have a working plot that I could visualize. We should probably standardize on a delimiter though, why is `everest` using `;`?